lib/FindTests.js

```diff
@@ -74,29 +74,27 @@ function resolveCliArgGlob(
       ? globRelativeToProjectRoot.replace(/\\/g, '/')
       : globRelativeToProjectRoot;
 
-  return glob
-    .sync(pattern, {
-      cwd: projectRootDir,
-      nocase: true,
-      absolute: true,
-      ignore: ignoredDirsGlobs,
-      // Match directories as well and mark them with a trailing slash.
-      nodir: false,
-      mark: true,
-    })
-    .flatMap((filePath) =>
-      filePath.endsWith('/') ? findAllElmFilesInDir(filePath) : filePath
-    );
+  return globSync(pattern, {
```

Comment:
Note to self: there a big comment above about glob@8 and Windows and backslashes. Try it out on Windows to see how it behaves in tinyglobby, and remove or update the code/comment?

Reply:
It’s needed: [`7814ff7` (#647)](https://github.com/rtfeldman/node-test-runner/pull/647/commits/7814ff7badeaf58024a150c5b045dfc259b47373)

Updated the comment: [`46ca4aa` (#647)](https://github.com/rtfeldman/node-test-runner/pull/647/commits/46ca4aa27f4104ac1d7e34acecb4304939e3df69)